mkdocs/docs/HPC/conda.md

```diff
@@ -0,0 +1,36 @@
+# Conda
+
+We do not recommend the use of conda environments on the HPC clusters. 
+This section explains why and highlights some common issues.
+
+## Performance and Optimization
+
+Conda's packages are pre-compiled binaries that are designed to work on a wide range of systems. 
+This means they are not optimized for the specific architecture of HPC clusters, 
+leading to potential performance drawbacks compared to modules compiled specifically for the HPC environment.
+
+HPC modules on the other hand, are compiled for the specific architecture of the cluster,
+and are optimized for performance.
+
+## Compatibility and Dependency
+
+Using Conda in conjunction with centrally installed modules can lead to conflicts and unexpected errors, 
+making it difficult to debug and manage dependencies.
```

Comment:
So users should install software to their conda env, even a central module is available.

Reply:
added in [9392906](https://github.com/hpcugent/vsc_user_docs/pull/649/commits/939290602752d9b5333fe1da4b2ed463c789b534)